src/sshagent/AgentSettingsPage.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ *  Copyright (C) 2017 Toni Spets <toni.spets@iki.fi>
```

Comment:
All new files need to include the standard KeePassXC team copyright (in can be in addition to your own):

```Copyright (C) 2017 KeePassXC Team <team@keepassxc.org>```